index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Google Font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Actor&display=swap" rel="stylesheet">
+    <!-- Custom Stylesheet -->
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Cinema</title>
+</head>
+
+<body>
+    <!-- Container  -->
+    <section class="container">
+
+        <!-- Header -->
+        <header>
+            <div>
+                <h1 class="nameOfActor">Bohemian <br> Rhapsody</h1>
+                <p class="category">Drama, Music</p>
+                <button class="description"><a href="pages/description.html">Description</a></button>
+            </div>
+        </header>
+
+        <!-- Wrapper -->
+        <section class="wrapper">
+            <!-- Multi Icons -->
+            <main class="multiIcons">
+                <div class="download divIcon"><a href="pages/download.html"><img src="images/download.svg" alt="Download Icon"></a>
+                </div>
+                <div class="favorite divIcon"><a href="#"><img src="images/favorite.svg" alt="Favorite Icon"></a>
+                </div>
+                <div class="Share divIcon"><a href="#"><img src="images/Share.svg" alt="Share Icon"></a> </div>
```

Comment:
you save the share icon with lowercase (share.svg) but you use it the uppercase (Share.svg) 

![Screenshot from 2021-06-26 15-00-26](https://user-images.githubusercontent.com/47992412/123512286-4a4a9000-d68f-11eb-8f8a-45c8f753a133.png)


![Screenshot from 2021-06-26 15-00-53](https://user-images.githubusercontent.com/47992412/123512294-56cee880-d68f-11eb-9eb0-e42a2b2a421a.png)

Reply:
Done